From 23d9dbfb3ad4a8d5d5997738e8b1436d076898e9 Mon Sep 17 00:00:00 2001
From: Eric Le Bihan <eric.le.bihan.dev@free.fr>
Date: Tue, 19 Apr 2016 10:49:20 +0200
Subject: [PATCH 1/3] Harmonize use of CROSS_PREFIX_* in mk/cfg/*.mk

Use CROSS_PREFIX_ in Makefile fragments for cross-compiling on Linux
wherever possible.

Signed-off-by: Eric Le Bihan <eric.le.bihan.dev@free.fr>
---
 mk/cfg/mips-unknown-linux-gnu.mk   | 9 +++++----
 mk/cfg/mipsel-unknown-linux-gnu.mk | 9 +++++----
 2 files changed, 10 insertions(+), 8 deletions(-)

diff --git a/mk/cfg/mips-unknown-linux-gnu.mk b/mk/cfg/mips-unknown-linux-gnu.mk
index 65b0877..0fe95de 100644
--- a/mk/cfg/mips-unknown-linux-gnu.mk
+++ b/mk/cfg/mips-unknown-linux-gnu.mk
@@ -1,8 +1,9 @@
 # mips-unknown-linux-gnu configuration
-CC_mips-unknown-linux-gnu=mips-linux-gnu-gcc
-CXX_mips-unknown-linux-gnu=mips-linux-gnu-g++
-CPP_mips-unknown-linux-gnu=mips-linux-gnu-gcc -E
-AR_mips-unknown-linux-gnu=mips-linux-gnu-ar
+CROSS_PREFIX_mips-unknown-linux-gnu=mips-linux-gnu-
+CC_mips-unknown-linux-gnu=gcc
+CXX_mips-unknown-linux-gnu=g++
+CPP_mips-unknown-linux-gnu=gcc -E
+AR_mips-unknown-linux-gnu=ar
 CFG_LIB_NAME_mips-unknown-linux-gnu=lib$(1).so
 CFG_STATIC_LIB_NAME_mips-unknown-linux-gnu=lib$(1).a
 CFG_LIB_GLOB_mips-unknown-linux-gnu=lib$(1)-*.so
diff --git a/mk/cfg/mipsel-unknown-linux-gnu.mk b/mk/cfg/mipsel-unknown-linux-gnu.mk
index 4dadfc2..9340c46 100644
--- a/mk/cfg/mipsel-unknown-linux-gnu.mk
+++ b/mk/cfg/mipsel-unknown-linux-gnu.mk
@@ -1,8 +1,9 @@
 # mipsel-unknown-linux-gnu configuration
-CC_mipsel-unknown-linux-gnu=mipsel-linux-gnu-gcc
-CXX_mipsel-unknown-linux-gnu=mipsel-linux-gnu-g++
-CPP_mipsel-unknown-linux-gnu=mipsel-linux-gnu-gcc
-AR_mipsel-unknown-linux-gnu=mipsel-linux-gnu-ar
+CROSS_PREFIX_mipsel-unknown-linux-gnu=mipsel-linux-gnu-
+CC_mipsel-unknown-linux-gnu=gcc
+CXX_mipsel-unknown-linux-gnu=g++
+CPP_mipsel-unknown-linux-gnu=gcc
+AR_mipsel-unknown-linux-gnu=ar
 CFG_LIB_NAME_mipsel-unknown-linux-gnu=lib$(1).so
 CFG_STATIC_LIB_NAME_mipsel-unknown-linux-gnu=lib$(1).a
 CFG_LIB_GLOB_mipsel-unknown-linux-gnu=lib$(1)-*.so
--
2.1.4

